src/main/java/org/fundacionjala/coding/david/SortInnerContent.java

```diff
@@ -0,0 +1,43 @@
+package org.fundacionjala.coding.david;
+
+import java.util.Arrays;
+
+/**
+ * This Class Word Reverse.
+ */
+public class SortInnerContent {
+
+    /**
+     * This method word reverse only content of Word.
+     *
+     * @param words is value number.
+     * @return a value word reverse.
+     */
+    public String sortInnerReverse(String words) {
```

Comment:
this method may be refactor 
something like this.
```java
public String sortInnerReverse(String words) {
        String delimiter = " ";
        String[] listWord = words.split(delimiter);
        StringJoiner listReverseWord = new StringJoiner(delimiter);
        for (String word : listWord) {
            int wordLength = word.length();
            if (wordLength >= 3) {
                String firstWord = word.substring(0, 1);

                String[] innerContent = word.substring(1, wordLength - 1).split("");
                Arrays.sort(innerContent);

                String lastWord = word.substring(wordLength - 1, wordLength);
                listReverseWord.add(firstWord
                        .concat(new StringBuilder(String.join("", innerContent)).reverse().toString())
                        .concat(lastWord));
            } else {
                listReverseWord.add(word);
            }
        }

        return listReverseWord.toString().trim();
    }
```

Don't declare unnecessary variables.
You can use a StringJoiner instead of a StringBuilder to avoid appending whitespaces.